### ISSUE TITLE
Accessibility should be checked using aXe plugin

### ISSUE THREAD

**Issue by koleaby4** [koleaby4/dom-designs#8]
As a web developer
I want to be sure 
That accessibility of the site has been tested by aXe chrome plugin

**Comment by koleaby4** [comment]
the only remaining vialotions are due to low contrast

![image](https://user-images.githubusercontent.com/12046340/58093772-1efb3a00-7bc7-11e9-9913-ad5a66363f8c.png)

These will be addressed as part of #23
